docs/features.md

```diff
@@ -1,24 +1,104 @@
-   all parts of the program code, under the GNU AGPLv3 license.
+[LibreTime](http://libretime.org/ "LibreTime homepage") is the open broadcast
+software for scheduling and remote station management. Web browser access to
+the station's media library, multi-file upload and automatic metadata import
+features are coupled with a collaborative online scheduling calendar and
+playlist management. The scheduling calendar is managed through an easy-to-use
+interface and triggers playout with sub-second precision.
 
-Screenshots
------------
+![](img/Screenshot540-Now_playing_250.png)
 
-![](manual/what-is-airtime/static/Screenshot540-Now_playing_250.png)
+LibreTime is intended to provide a solution for a wide range of broadcast
+projects, from community to public and commercial stations. The scalability of
+LibreTime allows implementation in a number of scenarios, ranging from an
+unmanned broadcast unit accessed remotely through the Internet, to a local
+network of machines accessing a central LibreTime storage system. LibreTime
+supports the playout of lossy compressed audio files in both MP3 and AAC
+formats and the open, royalty-free equivalent
+[Ogg Vorbis](http://www.vorbis.com/ "Ogg Vorbis homepage"). It also supports
+playout of lossless FLAC and WAV format audio files.
 
-![](static/Scheduler1.png)
+LibreTime manages the [Liquidsoap](http://savonet.sourceforge.net/) stream
+generator at the heart of the system. Liquidsoap generates streams from files
+in the LibreTime library and any remote input streams that you specify.
+Available stream output formats include Ogg Vorbis, Ogg Opus, MP3, and AAC. The
+library is indexed in a [PostgreSQL](http://www.postgresql.org/) database to
+enable searching. Live shows can be recorded automatically with
+[Ecasound](http://eca.cx/ecasound/ "Ecasound homepage"), using the sound card
+line input. News editors, DJs and station controllers can use LibreTime to
+build playlists or smart blocks and manage media files (upload, edit metadata,
+manage advertisements) at the station or via the Internet.
 
-![](static/Scheduler2.png)
+The scheduler in LibreTime has a calendar view, organized by months, weeks or
+days. Program editors can schedule playlists and shows here for their
+broadcast station. In some scenarios, the transmitter is situated outside the
+reach of the broadcaster and all program management has to be maintained
+through the web interface. Possible reasons for this scenario might be of a
+pragmatic nature (running many stations from one central office due to limited
+human resources) or an emergency (running a transmitter in a crisis area
+without putting staff at risk).
 
-![](static/Upload.png)
+LibreTime services
+----------------
+
+| Service | Description |
+|---------|-------------|
+| `airtime-analyzer` | Keeps track of files being added, renamed, moved or removed from storage, and reads their metadata using the Mutagen library. |
```

Comment:
The binary is called `airtime_analyzer`

Reply:
`libretime-analyzer` now I guess. It isn't the name of the binary, but people will work with the systemd unit more often anyway

Reply:
Still outstanding

Reply:
The file was updated with the correct information

Reply:
Ah right, thanks